tests/test_tomcat.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+import requests
+import tenacity
+from pytest_container import container_and_marks_from_pytest_param
+from pytest_container import DerivedContainer
+from pytest_container.container import ContainerData
+from pytest_container.container import ImageFormat
+
+from bci_tester.data import TOMCAT_CONTAINERS
+
+
+TOMCAT_WITH_SAMPLE = []
+
+for tomcat_ctr in TOMCAT_CONTAINERS:
+    ctr, marks = container_and_marks_from_pytest_param(tomcat_ctr)
+    TOMCAT_WITH_SAMPLE.append(
+        pytest.param(
+            DerivedContainer(
+                base=ctr,
+                forwarded_ports=ctr.forwarded_ports,
+                containerfile="""RUN cd $CATALINA_HOME/webapps; curl -O https://tomcat.apache.org/tomcat-10.1-doc/appdev/sample/sample.war
+HEALTHCHECK --interval=5s --timeout=5s --retries=5 CMD ["/usr/bin/curl", "http://localhost:8080/sample"]
```

Comment:
Add '-sf' to the parameters of curl to have erratic http responses actually throw a non zero exit code